apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -215,9 +215,26 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 
 	reqLogger.Info("getting security instance")
 
+	//get defined security cr from swagger
+	definedSecurity, checkSecuritykind := swagger.Extensions[securityExtension]
+	var securityName string
+
+	if checkSecuritykind {
+		rawmsg := definedSecurity.(json.RawMessage)
```

Comment:
Are we using tab to indent `go` src files? For java this is 4 space characters.

Reply:
This is added with go fmt (go formatting files) and it uses tabs for indentation.